uktrade/data-hub-frontend
# Show that there is a pending change request on the business details page

## Description of change

A box now appears on business details pages with pending changes to show that there is a pending change request that was submitted less than 20 days ago. This is acomplished by sending a query to the DNB service via the API that retrieves this information.

## Test instructions

Go to a DNB company and submit a change request. Once you have submitted the request the blue box will appear.

If you wish to test this locally, you will need to add the following environment variables to your API's `.env` file (the values are both in Vault):

- DNB_SERVICE_BASE_URL
- DNB_SERVICE_TOKEN

You will also need to have the VPN turned on as you need to access protected DNB service endpoints.

## Screenshots

![image](https://user-images.githubusercontent.com/42253716/89642125-13b72b80-d8ab-11ea-8b46-e978bb158bb8.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
